zerver/lib/event_queue.py

```diff
@@ -54,6 +56,7 @@ class ClientDescriptor(object):
     def __init__(self, user_profile_id, user_profile_email, realm_id, event_queue,
                  event_types, client_type_name, apply_markdown=True,
                  all_public_streams=False, lifespan_secs=0, narrow=[]):
+        # type: (int, str, int, EventQueue, List[str], str, Optional[bool], Optional[bool], Optional[int], Iterable[Sequence[text_type]]) -> None
```

Comment:
Are you sure `None` is a valid value for `apply_markdown`, `all_public_streams` and `lifespan_secs`?